tokio/src/runtime/builder.rs

```diff
@@ -728,6 +728,7 @@ impl Builder {
     /// # }
     /// ```
     #[cfg(all(not(loom), tokio_unstable))]
+    #[cfg_attr(docsrs, doc(cfg(tokio_unstable)))]
     pub fn on_task_spawn<F>(&mut self, f: F) -> &mut Self
```

Comment:
Please also add the docs blurb about this being unstable.
```rs
/// **Note**: This is an [unstable API][unstable]. The public API of this type
/// may break in 1.x releases. See [the documentation on unstable
/// features][unstable] for details.
///
/// [unstable]: crate#unstable-features
```
I would add it to `on_task_spawn`, `on_task_terminate`, and `TaskMeta`. Probably not to `MultiThreadAlt`.

Reply:
Done in 6b438c15